<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rank History</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,7">
   <property name="spacing">
    <number>7</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QScrollArea" name="pieConti">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>683</width>
          <height>380</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="eventBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="uniEdit">
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>32767</number>
         </property>
         <property name="placeholderText">
          <string>请输入希望查询的学校</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="uniBox"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="add">
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dlt">
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="uniList"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="lineRank">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1376</width>
        <height>535</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
